README.md

```diff
@@ -110,6 +110,7 @@ public enum MessageKind {
     case emoji(String) // TextMessageCell
     case audio(AudioItem) // AudioMessageCell
     case contact(ContactItem) // ContactMessageCell
+    case linkPreview(LinkItem) // LinkPreviewMessageCell
```

Comment:
Wondering if this should be released with a new major version (i.e 4.0.0) because it adds a new case to a public enum. Switches on enum types must be exhaustive so anyone doing a switch on a MessageKind without a `default` case (completely legal) will have their code break.

Reply:
Yeah, next version should be 4.0.0 anyways as we dropped iOS 9 and iOS 10 which is another breaking change. :)